parameters/dita-command-arguments.dita

```diff
@@ -431,6 +431,12 @@
           <pd>Verbose logging prints additional information to the console, including directory settings, effective
             values for Ant properties, input/output files, and informational messages to assist in troubleshooting.</pd>
         </plentry>
+        <plentry>
+          <pt>
+            <parmname>-no-color</parmname>
+          </pt>
+          <pd>By default, the DITA-OT outputs certain logging messages wrapped in special characters to get a colored output. If you prefer a clean output, you can set this parameter to deactivate the colored logging output.<codeblock>�[31m[move-meta] Failed to read file:/tmp/abc.dita: Failed to parse /tmp/abc.dita:Failed to read document: XML document structures must start and end within the same entity.�[0m</codeblock></pd>
```

Comment:
> If you prefer a clean output

This makes it sound like the default is somehow "dirty", but what's happening here is that your terminal doesn't support colored output. Will rephrase this accordingly.

While we're at it, lets sort this alphabetically along with the rest of the listing.